src/server/endpoints/covidcast_utils/model.py

```diff
@@ -272,7 +272,7 @@ def create_source_signal_alias_mapper(source_signals: List[SourceSignalPair]) ->
             continue
         # uses an alias
         alias_to_data_sources.setdefault(source.db_source, []).append(source)
-        if pair.signal == True:
+        if pair.signal:
```

Comment:
```suggestion
        if pair.signal is True:
```

im pretty sure you wanna have this test for `True`, because that is the `signal=*` (wildcard) indicator.  the alternative for `pair.signal` if it is not `True` would be a sequence of strings (a list of signals), which is "truthy" (as long as its not empty) and so it would fall into this block of the conditional.  the string sequence should go into the `else` block to be used as-is, instead of enumerating all the signals for that source.

Reply:
Good catch, applied!